### ISSUE TITLE
groovy.lang.MissingMethodException: No signature of method: io.restassured.internal.ContentParser.parse()

### ISSUE THREAD

**Issue by gmartinezsan** [rest-assured/rest-assured#920]
Hi,

I'm using the following code to validate the response of a get method:

  ResponseSpecBuilder builder = new ResponseSpecBuilder();
        builder.expectStatusCode(200);
        builder.expectBody("items.status.size()", is(1));
        ResponseSpecification responseSpec = builder.build();

        given()
            .spec(requestSpec)   
            .expect()
            .log().ifValidationFails()
            .spec(responseSpec)
            .when()
            .get("/status/");

When I run this test the following error is thrown:

groovy.lang.MissingMethodException: No signature of method: io.restassured.internal.ContentParser.parse() is applicable for argument types: (io.restassured.internal.RestAssuredResponseImpl, io.restassured.internal.ResponseParserRegistrar, io.restassured.config.RestAssuredConfig, java.lang.Boolean) values: [io.restassured.internal.RestAssuredResponseImpl@3d9f6567, io.restassured.internal.ResponseParserRegistrar@c055c54, ...]
Possible solutions: wait(), any(), grep()

I was expecting to have an assert failed message but I do not know why is it throwing this exception instead. Or if this is a limitation of the framework.

Any suggestions or hints are very welcome.

**Comment by johanhaleby** [comment]
I can't reproduce this, when I try I get this:

```java
java.lang.AssertionError: 1 expectation failed.
JSON path items.status.size() doesn't match.
Expected: is <1>
  Actual: null
```

which is what I expect in my case. What version of REST Assured are you using? Are you using maven/gradle? If not, have you really added all dependencies to the classpath? (see getting started guide)

**Comment by gmartinezsan** [comment]
Thanks for the quick reply, 
I'm not using maven or gradle yet. I'm doing the builds with ant and I have the following dependencies added to the class path of the application:

rest-assured-3.0.3.jar
junit-4.12.jar
commons-lang3-3.4.jar
commons-logging-1.2.jar
groovy-2.4.9.jar
groovy-json-2.4.9.jar
hamcrest-core-1.3.jar
hamcrest-library-1.3.jar
httpclient-4.5.2.jar
httpcore-4.4.4.jar
httpmime-4.5.1.jar
gson-2.8.0.jar

I saw that the json-schema-validator library is optional, so I did not add it.
I checked the getting started guide but did not see anything missing. I think the error could be in the dependencies, just notice that is says to put the rest assured dependency before the jUnit, which I did.
But still getting the exception.
